rtl/eclrtl/eclregex.cpp

```diff
@@ -468,65 +468,3 @@ ECLRTL_API void rtlDestroyUStrRegExprFindInstance(IUStrRegExprFindInstance * fin
 {
 }
 #endif // _USE_BOOST_REGEX or _USE_C11_REGEX
-
-#ifdef _USE_CPPUNIT
```

Comment:
Note - these tests are in eclrtl.cpp. They were supposed to be removed from here...